fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -512,14 +511,26 @@ private void handleShowDb() throws AnalysisException {
                     PrivPredicate.SHOW)) {
                 continue;
             }
-
             dbNameSet.add(db);
         }
 
         for (String dbName : dbNameSet) {
             rows.add(Lists.newArrayList(dbName));
         }
 
+        if (showDbStmt.getDb() != null) {
+            String db = showDbStmt.getDb();
+            MetadataMgr metadataMgr = GlobalStateMgr.getCurrentState().getConnectorMgr().getMetadataMgr();
```

Comment:
`Show databases from hive_catalog_1;` dql needs to fetch meta info from hms.If hms is down, may throw exeception here.